### PR TITLE
added crud notebook made with HTML, SCSS, JS Crud Operations, Local Storage.

### PR DESCRIPTION
**Pull request by UsamaBinKashif**
This is a open source code to the notes app which uses local storage of your browser which means your data is secured within your device. This web-app also includes of searching functionality.
### Functionality
- Add notes (title, note).
- Delete Notes.
- Search Notes.
- All your notes will be saved inside your browser local storage so other users can not see it.

### CODE REVIEW

**Comment by mudit023** [comment]
@UsamaBinKashif Please add a small video or a gifs of your project inyour readme file.

File: note-everything/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Note Everything.
+
+This is a open source code to the notes app which uses local storage of your browser which means your data is secured within your device. This web-app also includes of searching functionality.
+
+## Table of contents
+
+- [Overview](#overview)
+  - [Links](#links)
+  - [Screenshot](#screenshot)
```
Comment (mudit023) [inline note-everything/README.md:9]: Add a small video of it's working.

**Review by mudit023 — CHANGES_REQUESTED** [review]
- [ ] Add a small video of your template working in your readme file.
- [ ] Make use of CSS variables

File: note-everything/scss/main.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
+*::after,
+::before,
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #019267;
```
[the remark this answers is not in the document]
Comment (UsamaBinKashif) [reply note-everything/scss/main.css:10]: check sir.

**Review by mudit023 — APPROVED** [review]
LGTM! Thanks for contributing.

**Pull request merged**